packages/components/tag/src/tag.css

```diff
@@ -82,80 +82,56 @@
 	}
 }
 
-:host(:not([busy]):not([locked])[type='success']) {
-	& .container {
-		background-color: var(--ts-color-green);
-	}
+:host(:not([busy]):not([locked])[type='success']) .container {
```

Comment:
Can we simplify the selectors? I think we can just override the busy and locke state if needed.